.github/workflows/ci.yml

```diff
@@ -288,153 +289,6 @@ jobs:
-
-      - name: Restore ${{ needs.cargo-registry-cache.outputs.cache-key }} cache
-        uses: actions/cache@c64c572235d810460d0d6876e9c705ad5002b353 # v2.1.6
-        with:
-          path: |
-            ~/.cargo/registry/index
-            ~/.cargo/registry/cache
-            ~/.cargo/git/db
-          key: ${{ needs.cargo-registry-cache.outputs.cache-key }}
-
-      - name: Restore build artifacts
-        uses: actions/cache@c64c572235d810460d0d6876e9c705ad5002b353 # v2.1.6
-        with:
-          path: ${{ github.workspace }}/target
-          key: ${{ runner.os }}-${{ steps.rust-toolchain.outputs.rustc }}-${{ github.job }}-${{ matrix.target }}-${{ needs.cargo-registry-cache.outputs.cache-date }}-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: ${{ runner.os }}-${{ steps.rust-toolchain.outputs.rustc }}-${{ github.job }}-${{ matrix.target }}-${{ needs.cargo-registry-cache.outputs.cache-date }}-
-
-      - name: Build iOS library ${{ matrix.target }}
-        working-directory: ./xayn-ai-ffi-c
-        run: cargo build --target ${{ matrix.target }} -p xayn-ai-ffi-c
-
-      - name: Prepare lib for upload
-        run: |
-          dir=target/${{ matrix.target }}/debug
-          mv $dir/libxayn_ai_ffi_c.a $dir/libxayn_ai_ffi_c_${{ matrix.target }}.a
-
-      - name: Upload library artifacts
-        uses: actions/upload-artifact@27121b0bdffd731efa15d66772be8dc71245d074 # v2.2.4
-        with:
-          name: build-ios-${{ matrix.target }}-${{ github.sha }}
-          retention-days: 1
-          if-no-files-found: error
-          path: target/${{ matrix.target}}/debug/libxayn_ai_ffi_c_${{ matrix.target }}.a
-
-  test-wasm-libs:
-    name: test-wasm-libs
-    needs: [cargo-registry-cache, cargo-test]
-    runs-on: ubuntu-20.04
-    timeout-minutes: 20
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
-
-      - name: Install ${{ env.RUST_STABLE }} toolchain
-        id: rust-toolchain
-        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # v1.0.7
-        with:
-          profile: minimal
-          toolchain: ${{ env.RUST_STABLE }}
-          target: wasm32-unknown-unknown
-          default: true
-
-      - name: Restore ${{ needs.cargo-registry-cache.outputs.cache-key }} cache
-        uses: actions/cache@c64c572235d810460d0d6876e9c705ad5002b353 # v2.1.6
-        with:
-          path: |
-            ~/.cargo/registry/index
-            ~/.cargo/registry/cache
-            ~/.cargo/git/db
-          key: ${{ needs.cargo-registry-cache.outputs.cache-key }}
-
-      - name: Install wasm-pack
-        uses: ./.ci/install-wasm-pack
-
-      - name: Download data
-        run: sh download_data.sh
-
-      - name: Test WASM library
-        working-directory: ./xayn-ai-ffi-wasm
-        env:
-          RUSTFLAGS: ${{ env.OPT_TESTS_RUSTFLAGS }}
-        run: wasm-pack test --firefox --chrome --headless
-
   build-linux-lib:
```

Comment:
Can't we remove also this? This job builds the libraris to be used in flutter-build-example for android.

Reply:
Yes, but we still need to run the build scripts in order to be able to generate the ffi bindings otherwise the analyzer will fail. 
we could get the headers from `cargo check` and remove that job

Reply:
ah you are right, we need to keep it then because we are running the tests also there.